highfidelity/hifi
# Implements landscape mode for tablet as rotation instead of stretching [DO NOT MERGE]

Test plan:
- Open tablet, press Create button
- make sure Tablet rotated and everything works in Landscape mode
- close Create mode. Make sure Tablet returned back to portrait
